.github/workflows/ci.yml

```diff
@@ -8,8 +8,29 @@ on:
 
 env:
   FORCE_COLOR: 2
+  NODE_LINT: 14 # The Node.js version to run lint
```

Comment:
Can we add the `current` node version as a CI target and only run lint on that?
This way we don't need to update this when we drop supported versions.

Reply:
This doesn't work with Actions. They only support LTS aliases AFAICT.

Reply:
That's unfortunate 👍 Let's bump this to 16 then to defer this problem as long as possible.

Reply:
Oh right, OK.

Reply:
When you update to npm 7.x, and Node.js is LTS (sometime this October), you can change it :)